libraries/rush-lib/src/index.ts

```diff
@@ -84,6 +84,12 @@ export { IOperationRunner, IOperationRunnerContext } from './logic/operations/IO
 export { IExecutionResult, IOperationExecutionResult } from './logic/operations/IOperationExecutionResult';
 export { IOperationOptions, Operation } from './logic/operations/Operation';
 export { OperationStatus } from './logic/operations/OperationStatus';
+export { IBuildTimeRecord, _setBuildTimes } from './logic/operations/BuildTimePlugin';
+export {
+  IDependencyGraph,
+  dependencyAnalysis,
+  simulateBuildTime
```

Comment:
Do these functions need to be part of the public API of rush-lib?

Reply:
No they do not - I removed them.